src/cmap/auth/gssapi.ts

```diff
@@ -71,6 +71,7 @@ function makeKerberosClient(authContext: AuthContext, callback: Callback<Kerbero
   if ('kModuleError' in Kerberos) {
     return callback(Kerberos['kModuleError']);
   }
+  const { initializeClient } = Kerberos;
```

Comment:
What motivated pulling this out vs leaving it as `Kerberos.initializeClient`?

Reply:
Cus Typescript! tsd has its own version of TS which was a version newer and these lines were throwing errors. You'll notice the deps.ts file needed changes to make the types more accurate. Pulling out the function here after you've narrowed the type makes it exist properly inside the callback below. Where as Kerberos.initializeClient has the error that the property potentially doesn't exist. 

I can move the narrowing down into the callback too or just cast this problem away, open to diff solutions here

Reply:
I think it's fine - slightly less readable, but not so bad as to necessitate a change.